pkg/i18n/korean.go

```diff
@@ -88,6 +88,7 @@ func koreanTranslationSet() TranslationSet {
 		SureFixupThisCommit:              "Are you sure you want to 'fixup' this commit? It will be merged into the commit below",
 		SureSquashThisCommit:             "Are you sure you want to squash this commit into the commit below?",
 		Squash:                           "Squash",
+		SquashMerge:                      "SquashMerge",
```

Comment:
I think we should just leave non-english translation files alone: This particular example is not particularly useful.

Reply:
bruh, ok done